### ISSUE TITLE
Update first 'page' of content

### ISSUE THREAD

**Issue by jgclark** [chineham-repair-cafe/site-production#4]
We need to improve the header and first 'page'.

With the scroll that means that there are 4 overlapping pieces of text at one point:
![crc-messy-top-page@2x](https://github.com/chineham-repair-cafe/site-production/assets/1671754/05b48b5c-09cd-4456-a815-04f2f7b87923). To help, please:
- change the header bar background to be opaque not transparent
- remove the scroller
- I've done a tighter version of this logo to use in the left of this header area:
![CRC Logo Long (tight)](https://github.com/chineham-repair-cafe/site-production/assets/1671754/5c851c90-305f-40b9-b1a9-6fd907165fdb)

Our strapline is "Bin it? No way! Fix it!" not "Don't bin it, we can fix it". Please update.

We don't use the logo in 2 parts at 90 degrees to each other, which is shown at the moment.
Instead we use one of these two styles, with or without borders.
![CRC Logo (on orange)](https://github.com/chineham-repair-cafe/site-production/assets/1671754/e9fbda5e-7360-4348-841c-74ad461abe70)
![CRC Logo Long](https://github.com/chineham-repair-cafe/site-production/assets/1671754/89ae6db4-333b-4a33-8a75-0990bfd9dea7)

There's spare space on the bottom of this first 'page' on all device sizes, so I'd like to see a bold way of putting our next date and location there. I.e.:

OUR NEXT EVENT:
Saturday 18 NOVEMBER 2023, 10am-1pm
Christ Church Chineham, Reading Road, Basingstoke, RG24 8LT

Here are icons to use before the two lines
- ![calendar-icon@2x](https://github.com/chineham-repair-cafe/site-production/assets/1671754/084b930f-4c29-4051-bdc6-d67d531b6cd7)
- ![location-icon@2x](https://github.com/chineham-repair-cafe/site-production/assets/1671754/2f487a0b-d495-45a5-9f65-aff3b016dd4b)

(We will still keep the more detailed version in EVENTS & LOCATIONS below. Though change to LOCATION.)

**Comment by sibo999** [comment]
Hmm
Need abit of thought here. The large horizontal logo won't really work responsively which is why we originally went for the 'standard' right angle. I'll make this a bit smaller and get rid of it on small devices, plus reduce the font size which could achieve the objective.  I think the scroller should stay where it is as it's sticky at the top of the page and is 'attractive'. Should be amended to actual date though. 
I've amended the strapline text. 
Work in progress on hostgator Staging.

**Comment by shymega** [comment]
@sibo999 I'll hold off uploading anything to Hostgator, whilst you work on this. Please do let me know when I can upload, and I can upload my changes.

**Comment by sibo999** [comment]
Not sure which changes your referring to. But nothing should be broken. Goaheah and upload to staging.

Regards
Simon
m: 07984033914
________________________________

**Comment by shymega** [comment]
> Hmm Need abit of thought here. The large horizontal logo won't really work responsively which is why we originally went for the 'standard' right angle. I'll make this a bit smaller and get rid of it on small devices, plus reduce the font size which could achieve the objective. I think the scroller should stay where it is as it's sticky at the top of the page and is 'attractive'. Should be amended to actual date though. I've amended the strapline text. Work in progress on hostgator Staging.

@sibo999 Was referring to this. I'm not sure if you're working on the header side of things.

**Comment by sibo999** [comment]
I’ve done a bit of that. See how you think it works.

**Comment by sibo999** [comment]
Go with what we've got on staging.

**Comment by jgclark** [comment]
This is *not* complete from my perspective, at least on live (which from the messages I thought was updated?)

**Comment by shymega** [comment]
No, live hasn't been updated. Sorry for any confusion. For brevity reasons, I'll link to my comment [here](https://github.com/chineham-repair-cafe/chineham-site-production/issues/3#issuecomment-1853062292). I don't have access to live to change anything.

**Comment by jgclark** [comment]
I've now tried https://staging-chinehamrc.shymega.org.uk, and that doesn't
seem to have any 'first page' changes either.  What am I missing?

**Comment by sibo999** [comment]
I just tweaked some CSS opaqueness stuff as changing the image text needs a bit of testing.

**Comment by jgclark** [comment]
>
> I just tweaked some CSS opaqueness stuff as changing the image text needs
> a bit of testing.
>
> Message ID:
> ***@***.***
> com>
>
Sure … and that’s why we have a test site.  Please see what you can do, and
point me at the right place to test when ready. Thanks.
